spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -30,6 +30,22 @@
     end
   end
 
+  describe "GET Step schedule page" do
+    it "can only be accessed by users with Scheduling permissions" do
+      stub_user.permissions = ["signin", "GDS Editor", "Scheduling"]
+      get :schedule, params: { step_by_step_page_id: step_by_step_page.id }
+
+      expect(response.status).to eq(200)
+    end
+
+    it "cannot be accessed by users without Scheduling permissions" do
+      stub_user.permissions = ["signin", "GDS Editor"]
```

Comment:
This line isn't needed as these permissions are being set when `stub_user` is created.